### DISCUSSION TITLE
How can I check specific code that was published on May 21st?

### DISCUSSION THREAD

**Discussion by NarendraKuruva** [adobe/react-spectrum#6581] [General]
How can I check specific code that was published on May 21st?

@snowystinger

**Comment by snowystinger** [comment]
answered here https://github.com/adobe/react-spectrum/issues/6580#issuecomment-2179813294

**Reply by NarendraKuruva** [reply]
Got it. 

@snowystinger When will the latest code be published ?

**Reply by snowystinger** [reply]
There's no firm date, usually we try to be monthly, so should be soon.

**Reply by NarendraKuruva** [reply]
@snowystinger 

I've downloaded the source code from the release [From here](https://github.com/adobe/react-spectrum/releases/tag/%40adobe%2Freact-spectrum%403.35.0)

When I started my server I'm getting this error 
![image](https://github.com/adobe/react-spectrum/assets/102137869/47abd38a-c986-47af-a422-b7d42f9392c6)

Is this expected??

If not can you tell me how can I run the server without any errors?

**Reply by snowystinger** [reply]
Let's back up a little, why aren't you installing the version from npm? Why do you need to copy in the code instead?

If you need to make a small edit to the code while you wait for the next release, we recommend using `patch-package` or you can install a nightly version from npm.

**Comment by NarendraKuruva** [comment]
@snowystinger 

There are major UI changes in our combobox ui when compared to the default combobox ui

So we are copying the components & updating required styles & etc.... and making required changes.

**Reply by snowystinger** [reply]
Apologies, but I'm still not understanding why this is necessary. Would you be willing to put together a codesandbox demonstrating the kinds of changes you're making?
React Aria Components don't have any styles, you shouldn't need to copy them and edit our code in order to style them.

**Reply by NarendraKuruva** [reply]
Just to be on same page...
I'm copying `@react-spectrum/combobox` component & customising the styles.

not `React Aria Components`

@snowystinger

**Reply by snowystinger** [reply]
I see. You may want to switch to using RAC so you don't have this situation in the future.

That said, I'm not sure what is in that zip, but it seems to me like you should use git checkout for that commit hash. I can't guarantee that it will work either, depends on how far back in our history you are going. The further back you go, the less likely it is to work and the more research you may need to do to get an old build up and running.

**Reply by NarendraKuruva** [reply]
@snowystinger 

Checked **RAC**

The given example for asynchronous pagination is not handling the pagination & load more.

Our use case is mostly the async combobox

Here is the Video showing not handling the pagination
![Peek 2024-06-22 17-22](https://github.com/adobe/react-spectrum/assets/102137869/c468843d-ff59-4b74-80e1-e4d950203be7)

**Reply by snowystinger** [reply]
I see, we are working on updates to some of this right now so that we can swap the RSP implementation over to the RAC one. So hopefully in the future you'll be able to take advantage of the RAC version.

**Reply by NarendraKuruva** [reply]
@snowystinger 

Can you tell in how many weeks or months we can be able to use the RAC components??

**Reply by snowystinger** [reply]
I cannot, but the work has started being done here https://github.com/adobe/react-spectrum/pull/6494. Once that PR is done, we'll add it to listboxes as well, which should take care of combobox case
